docs/antrea-network-policy.md

```diff
@@ -281,7 +319,9 @@ where [uid] is the first 7 bits of hash value of the rule based on sha1 algorith
 If a policy contains duplicate rules, or if a rule name is same as the auto-generated
 name of some other rules in the same policy, it will cause a conflict,
 and the policy will be rejected.
-The example policy contains a single rule, which allows matched traffic on a
+A ClusterGroup name can be set in the `group` field of an ingress `from` section in place
+of stand-alone selectors to allow traffic from workloads/ipBlocks set in the ClusterGroup.
+The first example policy contains a single rule, which allows matched traffic on a
 single port, from one of two sources: the first specified by a `podSelector`
 and the second specified by a combination of a `podSelector` and a
 `namespaceSelector`.
```

Comment:
Maybe we should have this sentence as well: "The second example policy contains a single rule,  which allows matched traffic on multiple TCP ports (8000 through 9000 included, plus 6379) from all network endpoints selected by the "test-cg-with-db-selector" ClusterGroup."

I'm thinking we could do the same for appliedTo above and egress below.

Reply:
done